### ISSUE TITLE
tta CUESHEET support

### ISSUE THREAD

**Issue by ryuo** [tuffy/python-audio-tools#56]
I have only found one software so far that can produce whole cd rips with TTA. Python audio tools looks for this in the 'Cuesheet' while CUERipper embeds it within the 'CUESHEET' tag. Please fix the tag name. Or is there actually software that embeds it within the tag you are using now?

**Comment by tuffy** [comment]
The cuesheet tag isn't in the APEv2 or True Audio specifications at all, so it's going to have to be some sort of de-facto standard.  But WavPack uses APEv2 like TTA and it explicitly uses "Cuesheet" for the key name in its [specification.](http://www.wavpack.com/file_format.txt)  So it makes sense for two lossless formats using the same tagging format to use a non-standard tag like "Cuesheet" consistently.  Does CUERipper also use "CUESHEET" when embedding cuesheets in WavPack files?  If so, it may be a CUERipper bug.

"CUESHEET" itself is a strange choice considering how [all the other APEv2 tags are capitalized.](http://wiki.hydrogenaud.io/index.php?title=APE_key)  But I might be able devise an alias for it if necessary.

**Comment by ryuo** [comment]
Yes, it uses "CUESHEET" for both TTA and WV files. It appears to use the name "CUESHEET" for everything. FLAC, TTA, WV, APE, etc.

**Comment by tuffy** [comment]
That sounds good.  I've pushed out a fix which treats "CUESHEET" as an alias for "Cuesheet" in APEv2 tags.  Hopefully that'll take care of it.
